docs/intro/advanced-tutorial.rst

```diff
@@ -15,6 +15,7 @@ These are mainly achieved by **scrapy-poet** implementing **providers** for them
     * :class:`scrapy_poet.page_input_providers.HttpClientProvider`
     * :class:`scrapy_poet.page_input_providers.MetaProvider`
 
+.. _`intro-additional-requests`:
```

Comment:
I've added this to the checklist in https://github.com/scrapinghub/scrapy-poet/pull/68 so that we can cover all bases prior to a release. Updating the docs now would affect the other PRs awaiting to be merged.